its/ruling/src/integrationTest/resources/expected/jvm-framework-config/java-S2092.json

```diff
@@ -1,5 +1,5 @@
 {
-"jvm-framework-config-project:ruling/src/integrationTest/resources/sources/jvm-framework-config/S2092/S2092.yaml": [
+  "jvm-framework-config-project:ruling/src/integrationTest/resources/sources/jvm-framework-config/S2092/spring/S2092.yaml": [
```

Comment:
Please do not reformat json files in `its/ruling/src/integrationTest/resources/expected/` - it will make reviews harder and may cause merge conflicts

Reply:
Similar issue to the readme, thanks!